### PR TITLE
Create error alert

### PR DESCRIPTION
**Pull request by mdedys**

### CODE REVIEW

**Comment by mdedys** [comment]
This is complete. I just need to do the deployment setup

File: src/AppDispatcher.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Dispatcher } from 'flux';
+
+class AppDispatcher extends Dispatcher {
+
+	handleMappingsViewAction( action ) {
```
Comment (sebastianmihai) [inline src/AppDispatcher.js]: Remove "Mappings" from names

File: src/ErrorActions.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Actions } from './Constants.js';
+import AppDispatcher from './AppDispatcher.js';
+
+export default {
+
+	showConnErrorAlert( err ) {
+		AppDispatcher.handleMappingsBackgroundAction({
```
Comment (sebastianmihai) [inline src/ErrorActions.js]: Remove "Mappings" from names

File: src/AppDispatcher.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+class AppDispatcher extends Dispatcher {
+
+	handleMappingsViewAction( action ) {
+		this.dispatch({
+			source: 'MAPPINGS_VIEW_ACTION',
+			action: action
+		});
+	}
+
+	handleMappingsBackgroundAction( action ) {
```
Comment (sebastianmihai) [inline src/AppDispatcher.js]: I wonder why tabs are 8 spaces each here, but for my own pull request they are only 4 spaces each...

File: src/ErrorAlert.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+		if ( !this.state.errorHasHappened ) {
+			return null;
+		}
+
+		if ( this.state.errorType === ErrorTypes.SERVER_ERROR ) {
+			return (
+				<div className='server-error'>
+					<div>
+						<b>Oops!</b> We're having trouble connecting you. You might want to <a className='refresh-link' onClick={this.props.refresh}>refresh the page</a>, or try again later.
+					</div>
```
Comment (sebastianmihai) [inline src/ErrorAlert.js]: I'm not sure what's tripping GitHub up. The markup seems valid to me...

File: src/ErrorAlert.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+		if ( !this.state.errorHasHappened ) {
+			return null;
+		}
+
+		if ( this.state.errorType === ErrorTypes.SERVER_ERROR ) {
+			return (
+				<div className='server-error'>
+					<div>
+						<b>Oops!</b> We're having trouble connecting you. You might want to <a className='refresh-link' onClick={this.props.refresh}>refresh the page</a>, or try again later.
```
Comment (sebastianmihai) [inline src/ErrorAlert.js]: I tend to use <strong> instead of <b>, because it has more semantic meaning.

**Comment by sebastianmihai** [comment]
Looks good, aside from the small points I raised.

File: src/AppDispatcher.js (hunk 1 above)
Comment (mdedys) [inline src/AppDispatcher.js]: Done

File: src/ErrorActions.js (hunk 2 above)
Comment (mdedys) [inline src/ErrorActions.js]: Done

File: src/AppDispatcher.js (hunk 3 above)
Comment (mdedys) [inline src/AppDispatcher.js]: Fixed. I was missing the .editorconfig file that sets the indent_size. The actual spacing was 4 but github tends to use 8 when displaying unless specified.

File: src/ErrorAlert.js (hunk 5 above)
Comment (mdedys) [inline src/ErrorAlert.js]: Done

File: src/ErrorAlert.js (hunk 4 above)
Comment (mdedys) [inline src/ErrorAlert.js]: I think it might be the inline strong and inline anchor tags with the text.

Comment (mdedys) [inline src/ErrorAlert.js]: Actually it seems the `'` is tripping github out

**Pull request merged**